gridy/Sources/Presentations/PlanBoard/PlanBoardView.swift

```diff
@@ -348,20 +411,19 @@ extension PlanBoardView {
                                         text: viewStore.keyword,
                                         colorCode: PlanType.emptyPlanType.colorCode
                                     ))
-                                    viewStore.send(.listItemDoubleClicked(.emptyListItem, false))
+                                    viewStore.send(.dismissTextFieldOnList)
+                                }
+                                .onExitCommand {
+                                    viewStore.send(.dismissTextFieldOnList)
                                 }
-                                    .onExitCommand {
-                                        viewStore.send(.listItemDoubleClicked(.emptyListItem, false))
-                                    }
                             )
+                        // TODO: - 높이 수정
                             .frame(width: viewStore.listGridWidth - viewStore.columnStroke / 2, height: viewStore.lineAreaGridHeight - viewStore.rowStroke * 2)
```

Comment:
여기 원래 이렇게 작성되어있던 곳이긴한데 개행을 함께 부탁해요 !~!
```suggestion
                            .frame(
                            width: viewStore.listGridWidth - viewStore.columnStroke / 2, 
                            height: viewStore.lineAreaGridHeight - viewStore.rowStroke * 2
                            )
```

Reply:
도대체 왜...